packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -109,6 +118,11 @@ class CupertinoNavigationBar extends StatelessWidget implements ObstructingPrefe
   /// behind it.
   final Color backgroundColor;
 
+  /// The border of the navigation bar. By default renders a single pixel bottom border side.
+  ///
+  /// If a border is null, the nav bar will not display a border.
```

Comment:
here and maybe in other places, please use the full term "navigator bar" rather than "nav bar".

Reply:
Renamed other places to use full term "navigation bar".